src/utilities/transform-navigation-structure.ts

```diff
@@ -4,7 +4,17 @@ const navigationStructure = {
   components: {}
 }
 const nestedNavigationCategories = {
-  components: ['Overview', 'General', 'Layout', 'Media', 'Forms', 'Utilities']
+  components: [
+    'Overview',
```

Comment:
Should we sync these into the taxonomy document? So we have a reference point when we add components to know what to label things as?

Reply:
Yeah I think so 👍